scripts/travis-install

```diff
@@ -1,34 +1,55 @@
 #!/bin/bash
-set -e
-set -x
+set -e  # Exit on non-zero status
+set -x  # Print commands run
 export LC_CTYPE=en_US.UTF-8
 
-if [ "$TOXENV" == "py27" ]
+# Download ElasticSearch
+if [ ${INSTALL_ELASTICSEARCH:-0} -ne 0 ]
 then
-    sudo apt-get update -qq
-    sudo apt-get -y install build-essential libxml2-dev libxslt-dev libjpeg8 libjpeg8-dev libfreetype6 libfreetype6-dev zlib1g-dev sqlite3 tidy libtidy-dev libtidy-0.99-0 python-dev libffi-dev libssl-dev
-    sudo ln -s /usr/lib/`uname -i`-linux-gnu/libfreetype.so /usr/lib
-    sudo ln -s /usr/lib/`uname -i`-linux-gnu/libz.so /usr/lib
+    mkdir -p downloads
+    wget -q -O downloads/elasticsearch-$ES_VERSION.tar.gz $ES_DOWNLOAD_URL
```

Comment:
No, you can't. ElasticSearch keeps changing their downloads domain.  Here's a sample:

Version | URL
---|---
2.2.0|https://download.elasticsearch.org/elasticsearch/release/org/elasticsearch/distribution/tar/elasticsearch/2.2.0/elasticsearch-2.2.0.tar.gz
1.7.6|https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.7.6.tar.gz
5.3.0|https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.3.0.tar.gz